backend/census_historical_migration/sac_general_lib/audit_information.py

```diff
@@ -0,0 +1,150 @@
+from census_historical_migration.workbooklib.census_models.census import (
+    CensusGen22 as Gen,
+    CensusCfda22 as Cfda,
+    CensusFindings22 as Finding,
+)
+from census_historical_migration.base_field_maps import FormFieldMap, FormFieldInDissem
+from census_historical_migration.sac_general_lib.utils import (
+    _create_json_from_db_object,
+)
+import audit.validators
+
+DOLLAR_THRESHOLD = 750000
+
+mappings = [
+    FormFieldMap(
+        "dollar_threshold", None, FormFieldInDissem, DOLLAR_THRESHOLD, int

```

Comment:
There is. 

The draft dictionary (which needs to get published...)

https://federalist-35af9df5-a894-4ae9-aa3d-f6d95427c7bc.sites.pages.cloud.gov/preview/gsa-tts/fac-transition-site/jadudm/dictionaries/developers/v1_0_2/dictionary/

may help?

Reply:
Thanks for sharing this. I was struggling searching in googlesheets